geode-core/src/main/java/org/apache/geode/internal/lang/SystemPropertyHelper.java

```diff
@@ -71,6 +71,9 @@
 
   public static final String USE_HTTP_SYSTEM_PROPERTY = "useHTTP";
 
+  public static final String ENABLE_QUERY_RETRY_ON_PDX_SERIALIZATION_EXCEPTION =
```

Comment:
Better add some comments for the new system property. Add what is the default value for this property?